tests/test_wallet_regtest_mpp.py

```diff
@@ -59,19 +59,24 @@ async def _mint_pay_mpp(invoice: str, amount: int, proofs: List[Proof]):
             fee_reserve_sat=quote.fee_reserve,
             quote_id=quote.quote,
         )
+
     def mint_pay_mpp(invoice: str, amount: int, proofs: List[Proof]):
         asyncio.run(_mint_pay_mpp(invoice, amount, proofs))
 
     # call pay_mpp twice in parallel to pay the full invoice
-    t1 = threading.Thread(target=mint_pay_mpp, args=(invoice_payment_request, 32, proofs1))
-    t2 = threading.Thread(target=partial_pay_real_invoice, args=(invoice_payment_request, 32, 1))
+    t1 = threading.Thread(
+        target=mint_pay_mpp, args=(invoice_payment_request, 32, proofs1)
+    )
+    t2 = threading.Thread(
+        target=partial_pay_real_invoice, args=(invoice_payment_request, 32, 1)
+    )
 
     t1.start()
     t2.start()
     t1.join()
     t2.join()
 
-    assert wallet.balance <= 256 - 32
+    assert wallet.balance == 64
```

Comment:
```suggestion
    assert wallet.balance == 128 - 32
```
We have 128 in the wallet, but we only pay 32 from the wallet. The other 32 is paid from another node.